app/phone-list/phone-list.component.js

```diff
@@ -23,33 +23,33 @@ angular
           $scope.gridOptions.data = $scope.phones;
         });
 
+        /**
+         * We need to ensure that we have gridApi and the data needed.
+         * onRegisterApi is meant to register custom functions and does
+         * not call local functions.
+         */
+        $scope.$watchGroup(['gridApi', 'phones'], function (newValues, _) {
```

Comment:
`$watchGroup`: https://docs.angularjs.org/api/ng/type/$rootScope.Scope#$watchGroup

Reply:
`function (newValues, _)` We aren't using the "old" value, so we've used an underscore to indicate that we're not using it.